pkg/ccl/backupccl/schedule_pts_chaining.go

```diff
@@ -31,18 +31,59 @@ import (
 	pbtypes "github.com/gogo/protobuf/types"
 )
 
-// maybeUpdateSchedulePTSRecord is responsible for chaining of protected ts
-// records across scheduled backups. For a detailed outline of the scheme refer
-// to the comment above
-// ScheduledBackupExecutionArgs.ChainProtectedTimestampRecords.
+// This comment explains how scheduled backups chain protected timestamp
+// records.
```

Comment:
perhaps this is redundant...
Maybe drop or rephrase: "Protected ts chaining in scheduled backups"...?

Reply:
Dropped.